ledger/ledger.go

```diff
@@ -69,16 +68,25 @@ type Ledger struct {
 	headerCache heapLRUCache
 }
 
+// InitState structure defines blockchain init params
+type InitState struct {
+	InitBlocks   []bookkeeping.Block
+	InitAccounts map[basics.Address]basics.AccountData
+	GenesisHash  crypto.Digest
+}
+
 // OpenLedger creates a Ledger object, using SQLite database filenames
-// based on dbPathPrefix (in-memory if dbMem is true).  initBlocks and
-// initAccounts specify the initial blocks and accounts to use if the
+// based on dbPathPrefix (in-memory if dbMem is true). seed.initBlocks and
+// seed.initAccounts specify the initial blocks and accounts to use if the
 // database wasn't initialized before.
-func OpenLedger(log logging.Logger, dbPathPrefix string, dbMem bool, initBlocks []bookkeeping.Block, initAccounts map[basics.Address]basics.AccountData, genesisHash crypto.Digest) (*Ledger, error) {
+func OpenLedger(
+	log logging.Logger, dbPathPrefix string, dbMem bool, seed InitState, isArchival bool,
```

Comment:
don't refer to the InitState as seed; the usage of the term `seed` is already overloaded as is.
genesisInitState would be much cleaner.

Reply:
Done